pkg/configuration/reader/model.go

```diff
@@ -94,3 +95,8 @@ type Gateway struct {
 	Multiplier uint   `yaml:"multiplier,omitempty"`
 	Trench     string `yaml:"trench"`
 }
+
+type Range struct {
+	Start uint32 `yaml:"Start"`
```

Comment:
Case-mix; Start / end. start / end is better

Reply:
fixed in "[Identifer range on LB](https://github.com/Nordix/Meridio/pull/175/commits/18700b454dbec02c9176a81bbb3c5637c9660caf)"